### DISCUSSION TITLE
how to unpack release file?

### DISCUSSION THREAD

**Discussion by JackieZhuo** [zfl9/chinadns-ng#147] [Q&A] [answered]
i download release file from Assets, but don't know how to make it work.
could you please tell me how to unpack it?

**Comment by JackieZhuo** [comment]
下载下来就是这样的文件名chinadns-ng-x86_64，没有后缀名，请问这是什么类型的压缩包？要怎么打开或者解压缩？

**Comment by xtccc** [comment]
直接就是二进制吧
不是压缩包吧

**Reply by JackieZhuo** [reply]
这文件怎么在Windows上运行呢？chinadns-ng-x86_64我给他加个exe后缀也不能运行呀

**Reply by xtccc** [reply, marked as answer]
哦 windows 是吧, 该程序不支持在windows上运行
